picard/ui/__init__.py

```diff
@@ -33,9 +33,21 @@
 )
 
 from picard import config
+from picard.const.sys import (
+    IS_MACOS,
+    IS_WIN,
+)
 from picard.util import restore_method
 
 
+if IS_MACOS:
+    FONT_FAMILY_MONOSPACE = 'Menlo'
+if IS_WIN:
```

Comment:
Shouldn't be `elif` ? Because on MacOS, `FONT_FAMILY_MONOSPACE` will be set to `Monospace` in `else` part. Or I miss something.

Reply:
Looks like it should be `elif` to me too.

Reply:
Yes, absolutely. I'll change that. 

If we have this set here explicitly, we also could remove the workaround of setting a font substitution on macOS with `QtGui.QFont.insertSubstitution('Monospace', 'Menlo')` (see https://github.com/metabrainz/picard/blob/master/picard/tagger.py#L231). We only need one or the other. I tend to remove the font substitution and rely on the explicit font definition, what do you think?

Btw, I also tried the font substitution trick on Windows and it did not deliver good results. The font getting used looked horrible. Hence the explicit definition.